c/include/libsbp/edc.h

```diff
@@ -15,11 +15,13 @@
 
 #include <libsbp/common.h>
 
+#include "sbp_export.h"
```

Comment:
I just had a little go and this doesn't work when libsbp is installed to the filesystem:

```
> cat test.c
#include <libsbp/sbp.h>

int main(void) { return 0; }

> cc test.c -o test -Iinclude -Llib -lsbp
In file included from include/libsbp/sbp.h:17,
                 from test.c:1:
include/libsbp/legacy/api.h:18:10: fatal error: sbp_export.h: No such file or directory
   18 | #include "sbp_export.h"
      |          ^~~~~~~~~~~~~~
compilation terminated.

```

Reply:
I've since removed the use of `sbp_export.h` and placed everything in `common.h`. I've also added a new stage to all the builds which will consume the installed SBP library to catch any issues in the future that might arise from this.